### PR TITLE
Add links to Elastic APM implementations

### PR DESCRIPTION
**Pull request by felixbarny**
Some links are still missing, so this is currently just a draft.

Is the best practice to have permalinks to the currently most recent version?

### CODE REVIEW

File: implementations.md [hunk 1]
```diff
@@ -7,7 +7,13 @@ This page contains an alphabetical list of projects and vendors that currently i
...
 
 **Implementations:**
-[Node.js](https://github.com/elastic/node-traceparent).
+[.NET](https://github.com/elastic/apm-agent-dotnet/blob/700754909b1ac522796294b99adcc98063efcf42/src/Elastic.Apm/DistributedTracing/TraceParent.cs),
+[Go](),
+[Java](https://github.com/elastic/apm-agent-java/blob/e4cdde0b860ff37ea57e0ca083c62b319c0ee940/apm-agent-core/src/main/java/co/elastic/apm/agent/impl/transaction/TraceContext.java),
+[Node.js](https://github.com/elastic/node-traceparent),
+[Python](https://github.com/elastic/apm-agent-python/blob/50dce143ae15f6c592a70cb858a8c4721dd80ef5/elasticapm/utils/disttracing.py),
+[Ruby](https://github.com/elastic/apm-agent-ruby/blob/b68f1f12ae48a5c6e757241c65de97a98488ee6a/lib/elastic_apm/trace_context.rb),
+[JavaScript/RUM]()
```
Comment (hmdhk) [inline implementations.md]:
```suggestion
[JavaScript - Real User Monitoring](https://github.com/elastic/apm-agent-rum-js)
```

File: implementations.md [hunk 2]
```diff
@@ -7,7 +7,13 @@ This page contains an alphabetical list of projects and vendors that currently i
 **Website:** [https://elastic.co/products/apm](https://elastic.co/products/apm)
 
 **Implementations:**
-[Node.js](https://github.com/elastic/node-traceparent).
+[.NET](https://github.com/elastic/apm-agent-dotnet/blob/700754909b1ac522796294b99adcc98063efcf42/src/Elastic.Apm/DistributedTracing/TraceParent.cs),
+[Go](),
```
Comment (SergeyKanzhelev) [inline implementations.md]: this is missing

Comment (felixbarny) [inline implementations.md]:
```suggestion
[Go](https://github.com/elastic/apm-agent-go/blob/0365ba99e57221bfa884579c69cbcc0a16c2ded7/tracecontext.go),
```

**Comment by felixbarny** [comment]
Could someone revalidate on https://labs.w3.org/repo-manager/pr/id/w3c/trace-context/358, I have linked my accounts now.

File: implementations.md (hunk 2 above)
Comment (axw) [reply implementations.md]: https://github.com/elastic/apm-agent-go/blob/master/module/apmhttp/traceheaders.go is probably the more interesting part (HTTP header parsing).

**Comment by mtwo** [comment]
Hmm ... it's still failing after I revalidate

**Review by SergeyKanzhelev — APPROVED** [review]

**Comment by SergeyKanzhelev** [comment]
Marked as non substantive for IPR from ash-nazg.

**Review by mtwo — APPROVED** [review]

File: implementations.md [hunk 3]
```diff
@@ -7,7 +7,13 @@ This page contains an alphabetical list of projects and vendors that currently i
 **Website:** [https://elastic.co/products/apm](https://elastic.co/products/apm)
 
 **Implementations:**
-[Node.js](https://github.com/elastic/node-traceparent).
+[.NET](https://github.com/elastic/apm-agent-dotnet/blob/700754909b1ac522796294b99adcc98063efcf42/src/Elastic.Apm/DistributedTracing/TraceParent.cs),
+[Go](https://github.com/elastic/apm-agent-go/blob/0365ba99e57221bfa884579c69cbcc0a16c2ded7/tracecontext.go),
```
Comment (felixbarny) [inline implementations.md]:
```suggestion
[Go](https://github.com/elastic/apm-agent-go/blob/0e868bf43005f3f5b3786101960137d7c8760361/module/apmhttp/traceheaders.go),
```

**Review by plehegar — APPROVED** [review]

**Pull request merged**